docs/book/src/getting-started.md

```diff
@@ -45,7 +45,7 @@ An Azure Service Principal is needed for deploying Azure resources. The below in
   ```
 
   5. Create an Azure Service Principal by running the following command or skip this step and use a previously created Azure Service Principal.
-  NOTE: the "owner" role is required to be able to create role assignments for [system-assigned managed identity](vm-identity.md).
+  NOTE: the "owner" role is required to be able to create role assignments for system-assigned managed identity.
```

Comment:
When I looked at this article, I thought it was misleading since it was specific to self-managed. I also plan to entirely overhaul the getting started, so I thought was best to just remove the link.